index.html

```diff
@@ -1229,7 +1230,96 @@ <h1 class="article-title"><a href="">Article Title</a></h1>
+      <div class="row flex-spaces child-borders">
+        <a href="#modal-1" class="paper-btn margin">Open Modal!</a>
+      </div>
+      <div class="modal row flex-spaces" id="modal-1">
+        <div class="modal-body">
+          <a href="#modals" class="btn-close">x</a>
+          <h4 class="modal-title">Modal Title</h4>
+          <h5 class="modal-subtitle">Modal Subtitle</h5>
+          <p class="modal-text">This is an example of modal which is implemented with pure CSS! :D</p>
+          <a href="#modals"><button>Nice!</button></a>
+        </div>
+      </div>
+      <div class="docs">
+          <pre><code>&lt;div class="row flex-space child-borders"&gt;
+  &lt;a href="modal-1" class="paper-btn margin"&gt;Open Modal!&lt;/a&gt;
+&lt;/div&gt;
+&lt;div class="modal row flex-space" id="modal-1"&gt;
+  &lt;div class="modal-body"&gt;
+    &lt;a class="btn-close" href="#modals"&gt;x&lt;/a&gt;
+    &lt;h4 class="modal-title"&gt;Modal Title&lt;/h4&gt;
+    &lt;h5 class="modal-subtitle"&gt;Modal Subtitle&lt;/h5&gt;
+    &lt;p class="modal-text"&gt;This is an example of modal which is implemented with pure CSS! :D&lt;/p&gt;
+    &lt;button&gt;Nice!&lt;/button&gt;
+  &lt;/div&gt;
+&lt;/div&gt;</code></pre>
+        </div>
 
+      <h4>Modal with title, text and links</h4>
+      <div class="row flex-spaces child-borders">
+        <a href="#modal-2" class="paper-btn margin">Another Modal!</a>
+      </div>
+      <div class="modal row flex-spaces" id="modal-2">
+        <div class="modal-body">          
+          <a href="#modals" class="btn-close">x</a>
+          <h4 class="modal-title">Modal Title</h4>
+          <h5 class="modal-subtitle">Modal Subtitle</h5>
+          <p class="modal-text">This is an example of modal which is implemented with pure CSS! :D</p>
+          <a class="modal-link" href="#modals">OK</a>
+          <a class="modal-link" href="#modals">Close</a>
+        </div>
+      </div>
+      <div class="docs">
+          <pre><code>&lt;div class="row flex-space child-borders"&gt;
+  &lt;a href="modal-1" class="paper-btn margin"&gt;Open Modal!&lt;/a&gt;
+&lt;/div&gt;
+&lt;div class="modal row flex-space" id="modal-1"&gt;
+  &lt;div class="modal-body"&gt;
+    &lt;a class="btn-close" href="#modals"&gt;x&lt;/a&gt;
+    &lt;h4 class="modal-title"&gt;Modal Title&lt;/h4&gt;
+    &lt;h5 class="modal-subtitle"&gt;Modal Subtitle&lt;/h5&gt;
+    &lt;p class="modal-text"&gt;This is an example of modal which is implemented with pure CSS! :D&lt;/p&gt;
+    &lt;a class="modal-link" href="#modals"&gt;OK&lt;/a&gt;
+    &lt;a class="modal-link" href="#modals"&gt;Close&lt;/a&gt;
+  &lt;/div&gt;
+&lt;/div&gt;</code></pre>
+        </div>
+
+      <h4>Paper border</h4>
+      <div class="row flex-spaces child-borders">
+        <a href="#modal-3" class="paper-btn margin">Paper Modal!</a>
+      </div>
+      <div class="modal row flex-spaces" id="modal-3">
+        <div class="modal-body border">
```

Comment:
I'm personally not crazy about the look of a paper-border with modals. I think modals could represent a digital sticky note which would have sharp edges. The default styles look good!

Reply:
Should we remove the 3rd modal then?

Reply:
Please!